torchx/distributed/__init__.py

```diff
@@ -16,8 +16,11 @@
 
 import torch
 import torch.distributed as dist
+from torch.distributed.distributed_c10d import _get_default_group
 from typing_extensions import Literal
 
+from torchx.util.cuda import has_cuda_devices
+
 log: logging.Logger = logging.getLogger(__name__)
 
 
```

Comment:
For local_rank should we be returning os.environ["LOCAL_RANK"] if dist isn't initialized to match the new behavior of the other methods? Not sure if that was an oversight. 

Line 40 since I can't directly comment on that via GitHub ui

Reply:
yeah good point. Fixed `local_rank()` to return `LOCAL_RANK` if set, otherwise 0.